pyradiosky/tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+
+import pytest
+
+
+try:
+
+    import lunarsky     # noqa
+    hasmoon = True
+
+except(ImportError):
+    hasmoon = False
+reason = 'lunarsky is not installed, skipping tests that require it.'
+skip_if_no_moon = pytest.mark.skipif(not hasmoon, reason=reason)
```

Comment:
There's actually a pytest builtin called `importorskip` that you can call at the top of the tests that might be a little simpler than this. You can see how I used it for `astropy_healpix` in PR #44

Reply:
That looks useful! I've switched to `importorskip` for these tests.